Source/Light.swift

```diff
@@ -10,24 +10,40 @@ public struct Light: Equatable, CustomStringConvertible {
 	public let power: Bool
 	public let brightness: Double
 	public let color: Color
+	public let productInformation: ProductInformation?
```

Comment:
I would have preferred to see this called `productInfo` since that would feel closer to Cocoa (Eg: `userInfo`).